lumicks/pylake/correlated_stack.py

```diff
@@ -340,7 +340,7 @@ def select_frame(event):
 
         fig.canvas.mpl_connect('button_press_event', select_frame)
 
-    def export_tiff(self, file_name, roi):
+    def export_tiff(self, file_name, roi=None):
```

Comment:
I realized I had forgotten to put in the default value halfway through the 2nd commit. I think this can all be squashed though because it's really just one feature with a bunch of testing